cli/commandparser.ts

```diff
@@ -34,7 +36,7 @@ export interface Command {
 
 export interface ParsedCommand {
     name: string;
-    arguments: string[];
+    args: string[];
```

Comment:
why this renaming?

Reply:
arguments is a banned word.

Reply:
i don't understand why.

Reply:
Do not use banned terms: caller, callee, eval, arguments. These terms refer to functions or properties that should not be used, so it is best practice to simply avoid them.